docs/samples/v1beta1/triton/torchscript/README.md

```diff
@@ -333,24 +333,24 @@ The transformer does not enforce a specific schema like predictor but the genera
 {
   "instances": [
     {
-      "image": { "b64": "aW1hZ2UgYnl0ZXM=" },
+      "image_bytes": { "b64": "aW1hZ2UgYnl0ZXM=" },
       "caption": "seaside"
     },
     {
-      "image": { "b64": "YXdlc29tZSBpbWFnZSBieXRlcw==" },
```

Comment:
This keyword is needed at https://github.com/kserve/kserve/blob/master/docs/samples/v1beta1/triton/torchscript/image_transformer_v2/image_transformer_v2.py#L31

Reply:
These `b64` bytes actually crashes https://github.com/kserve/kserve/blob/master/docs/samples/v1beta1/triton/torchscript/image_transformer_v2/image_transformer_v2.py#L32, throwing error `OSError: cannot identify image file`. A random guess is Pillow updated its code and try to validate if bytes are image files. Tested with `b64` bytes from real images could work. But the strings are much longer.